Makefile

```diff
@@ -3,7 +3,7 @@ PORT ?= 3128
 
 SHELL = /bin/bash
 
-DESTDIR ?= /
+PREFIX ?= /usr
```

Comment:
"Packages should never be installed to /usr/local" .. this is clearly stated in arch linux packaging standards. If this is an incompatibility between distros, then we should override it for debian-install.

As for the `.service` file, systemd always needs absolute paths. We should never assume that a specific script (defining PATH) will run before the service. So, templating is the way to go.